cvat-ui/src/components/analytics-page/analytics-performance.tsx

```diff
@@ -0,0 +1,222 @@
+                            }
+                        }
+
+                        const leftName = transform.binary.left;
+                        const rightName = transform.binary.right;
+                        if (Object.hasOwn(dataseries, leftName) && Object.hasOwn(dataseries, rightName)) {
+                            dataseries[transform.name] = dataseries[leftName].map((left, i) => {
+                                const right = dataseries[rightName][i];
+                                if (typeof left.value === 'number' && typeof right.value === 'number') {
+                                    return {
+                                        value: operator(left.value, right.value),
+                                        date: left.date,
+                                    };
+                                }
+                                return {
+                                    value: 0,
+                                    date: left.date,
+                                };
+                            });
+                            delete dataseries[leftName];
+                            delete dataseries[rightName];
+                        }
+                    }
+                });
+
+                let colorIndex = -1;
+                const datasets = Object.entries(dataseries).map(([key, series]) => {
+                    let label = key.split('_').join(' ');
+                    label = label.charAt(0).toUpperCase() + label.slice(1);
+
+                    const data: number[] = series.map((s) => {
+                        if (typeof s.value === 'number') {
+                            return s.value as number;
+                        }
+
+                        if (typeof s.value === 'object') {
+                            return Object.keys(s.value).reduce((acc, k) => acc + s.value[k], 0);
+                        }
+
+                        return 0;
+                    });
+
+                    colorIndex = colorIndex >= colors.length - 1 ? 0 : colorIndex + 1;
+                    return {
+                        label,
+                        data,
+                        backgroundColor: colors[colorIndex],
+                    };
+                });
+                layout.push({
+                    i: name,
+                    h: 1,
+                    w: 2,
+                    x: 0,
+                    y: histogramCount,
+                });
+                histogramCount += 1;
+                return ({
+                    view: (
+                        <HistogramView
+                            datasets={datasets}
+                            labels={dateLabels}
+                            title={entry.title}
+                            key={name}
+                        />
+                    ),
+                    key: name,
+                });
+            }
+            default: {
+                throw Error(`View type ${entry.defaultView} is not supported`);
```

Comment:
```suggestion
                throw new Error(`View type ${entry.defaultView} is not supported`);
```

Reply:
Probably UI should not fail if there is a not supported view. Warning would be enough, I believe